README.md

```diff
@@ -1,47 +1,118 @@
+  - [Running a specific tests](#running-a-specific-tests)
+  - [Adding tests](#adding-tests)
+    - [Zemu](#zemu)
+    - [Update binaries](#update-binaries)
+- [Contributing](#contributing)
+
+
+</details>
+
+## About the project
+
+Ethereum wallet application framework for Nano S and Nano X.  
+Ledger Blue is not maintained anymore, but the app can still be compiled for this target using the branch [`blue-final-release`](https://github.com/LedgerHQ/app-ethereum/tree/blue-final-release).
+
+## Documentation
 
 This app follows the specification available in the `doc/` folder.
 
 To compile it and load it on a device, please check out our [developer portal](https://developers.ledger.com/docs/nano-app/introduction/).
 
-# Plugins
+### Plugins
 
-This app support external plugins. More info in [doc/ethapp_plugin.asc](https://github.com/LedgerHQ/app-ethereum/blob/master/doc/ethapp_plugins.asc). If you wish to have a look at an existing plugin, feel free to check out the [Boilerplate plugin](https://github.com/LedgerHQ/app-plugin-boilerplate).
+We have the concept of plugins in the ETH app.  
+Find the documentations here:  
+- [Blog Ethereum plugins](https://blog.ledger.com/ethereum-plugins/)
+- [Ethereum application Plugins : Technical Specifications](https://github.com/LedgerHQ/app-ethereum/blob/master/doc/ethapp_plugins.asc)
+- [Plugin guide](https://hackmd.io/300Ukv5gSbCbVcp3cZuwRQ)
+- [Boilerplate plugin](https://github.com/LedgerHQ/app-plugin-boilerplate)
 
-# Testing
+## Testing
 
 Testing is done via the open-source framework [zemu](https://github.com/Zondax/zemu).
 
-## Running tests
+### Requirement
 
-First [install yarn](https://classic.yarnpkg.com/en/docs/install/#debian-stable).
-Open `tests/build_local_test_elfs.sh` and add your BOLOS SDKs path to `NANOS_SDK` and `NANOX_SDK`.
-This helper script will build the applications required by the test suite and move them at the right place.
-```
-cd tests
-./build_local_test_elfs.sh
-```
-Then you can install the project by simply running:
-```
-cd ..
-make test
-```
-This will run `make install_tests` and `make run_tests`
+- [nodeJS](https://github.com/nvm-sh/nvm)
+- [yarn](https://classic.yarnpkg.com/lang/en/docs/install/#debian-stable)
+- [build environnement](https://github.com/LedgerHQ/ledger-app-builder/blob/master/Dockerfile)
 
-To run a specific tests (here the send test):
-```
-cd tests
-jest --runInBand --detectOpenHandles src/send.test.js
-```
+#### Build the applications required by the test suite  
+
+1. Add your BOLOS SDKs path to:
+    - `NANOS_SDK` and `NANOX_SDK`
+
+2. Go to the `tests` folder and run `./build_local_test_elfs.sh`
+    - ```sh
+        cd tests
+        # This helper script will build the applications required by the test suite and move them at the right place.
+        yarn install
+        ./build_local_test_elfs.sh
+      ```
+
+### Running all tests
+#### With Makefile
+
+1. Then you can install the project by simply running on the `root` of the repo:
```

Comment:
"install the project" or "running the tests"?

Reply:
For running the project you will install the project :)

Reply:
Indeed installing is implied. It just seem strange to have a statement "Then you can install the project" when the title and the command are about running the tests.

Reply:
I replace by this:

Then you can **install and run tests** by simply running on the `root` of the repo: